commands/general/help.js

```diff
@@ -1,55 +1,120 @@
 const Command = require(`${process.cwd()}/base/Command.js`);
-
+const {MessageEmbed} = require("discord.js");
+const perpage = 10;
+/*
+  The HELP command is used to display every command's name and description
+  to the user, so that he may see what commands are available. The help
+  command is also filtered by level, so if a user does not have access to
+  a command, it is not shown to them. If a command name is given with the
+  help command, its extended help is shown.
+*/
 class Help extends Command {
   constructor(client) {
     super(client, {
       name: "help",
-      description: "Displays the commands for your level.",
-      usage: "help [command]",
-      category: "System",
-      extended: "This command will display all available commands for your permission level, with the additonal option of getting per command information when you run 'help <command name>'.",
-      hidden: true,
-      aliases: ["h", "halp", "commands"]
+      description: "Get help on a command, command category, or a setting",
+      category: "Utility",
+      enabled:true,
+      guildOnly:false,
+      usage: "help <category/command/setting> [page-num]",
+      aliases: ["halp","h"],
+      permLevel: "User",
+      examples:["moderation","mute"]
     });
   }
 
   async run(message, args, level) {
-    const settings = message.settings;
+  // If no specific command is called, show all filtered commands.
+    const client = this.client;
+    const embed = new MessageEmbed();
+    // Load guild settings (for prefixes and eventually per-guild tweaks)
+    const settings = message.guild ? await this.client.settings.get(message.guild.id) : this.client.config.defaultSettings;
```

Comment:
Why remove the `settings` as we pass that via the [message event](https://github.com/NotAWeebDev/Misaki/blob/Yukine-Refactor/events/message.js#L13-L15), you're just duplicating code.